osl/interface_linux.go

```diff
@@ -130,6 +130,28 @@ func (n *networkNamespace) Interfaces() []Interface {
 	return ifaces
 }
 
+func (i *nwIface) SetSecondaryIP(ip *net.IPNet, add bool) error {
```

Comment:
Thanks, yes this is an update requirement which cannot be satified by current API/flow. To this regard though I am suggesting to follow a different approach in my other comment https://github.com/docker/libnetwork/pull/1501#discussion_r82470811 if feasible.